app/assets/javascripts/discourse/views/topic-list-item.js.es6

```diff
@@ -10,8 +10,17 @@ export default Discourse.View.extend(StringBuffer, {
        ],
 
   unboundClassNames: function(){
-    var classes = [];
     var topic = this.get('topic');
+    var classes = [];
+
+    if(topic.get('category')){
+      var category,categorySlug,categoryClass;
+      category = topic.get('category');      
+      categorySlug = category.get('slug');
+      categoryClass = 'category-' + categorySlug;
+
+      classes.push(categoryClass);
```

Comment:
Haha okay, made that change too. Thanks for the advice! I generally am not writing javascript...ever.. anymore, so every comment helps me